cuaca/api.py

```diff
@@ -162,29 +147,24 @@ def datatypes(self):
         GET a list of data types from API - to interpret the results from data
         endpoint
         """
-        url = "{}/{}".format(self.end_point, "datatypes")
+        url = "/".join(self.end_point, "datatypes")
         return self.call_api(url)
 
     def call_api(self, url, params={}, metadata=False):
-        """
-        A utility fuction to make API call easy, wrap around requests
-        """
-        k = "{}{}".format(url, str(params))
-        if k in self.cache:
+        """Wrapper to provide easy access to API call."""
+        k, now = "{}{}".format(url, str(params)), datetime.datetime.now()
```

Comment:
I think I might change the "{}" to query. I think this library can just use urllib straight away. Just a few more lines of code rather than one big chunk of dependency.